.github/workflows/build.yml

```diff
@@ -25,6 +25,12 @@ jobs:
       - name: Build packages
         run: yarn build
 
+      - name: Check bundle size
+        uses: jackyef/bundlewatch-gh-action@0.2.0
+        with:
+          bundlewatch-github-token: ${{ secrets.BUNDLEWATCH_GITHUB_TOKEN }}
+          bundlewatch-config: ../../.bundlewatchrc.json
```

Comment:
@rabelloo do you prefer this over config in `package.json`?

Reply:
P.S. This is breaking on CI, maybe just `.bundlewatchrc.json` works?

Reply:
Yeap. I forgot that it looks it up from the root of the project, not where the workflow is located. Should be working fine now.

Reply:
yup this looks great 💪🏻